src/neo/SmartContract/Native/Tokens/Nep11AccountState.cs

```diff
@@ -0,0 +1,21 @@
+using Neo.VM;
+using Neo.VM.Types;
+using System.Numerics;
+
+namespace Neo.SmartContract.Native.Tokens
+{
+    public class Nep11AccountState : IInteroperable
```

Comment:
Perhaps this can be merged with `Nep5AccountState.cs`, they are, in principle, as they are right now, the same, right?
Thus they could share the inheritance.